components/meta-packages/install-types/includes/minimal

```diff
@@ -195,7 +195,6 @@ depend type=require fmri=system/library/storage/scsi-plugins
 depend type=require fmri=system/man
 depend type=require fmri=system/network
 depend type=require fmri=system/network/nis
-depend type=require fmri=system/network/routing
 depend type=require fmri=system/prerequisite/gnu
```

Comment:
I have no idea.   The idea behind removing it from the minimal group is , so that it is not always installed (as part of the minimal group).   The fact that it is in "minimal" seems not logical as there may be setups that do not need this software so it is not minimal in the sense of absolutely required (I think).

Reply:
I have closed this PR but can reopen it later, perhaps I should first do more tests to build with distribution constructor images and see result of the change.   In any case I have opened bugs.illumos.org issues for these modifications and a request to split NPD neighbour ipv6 discovery from routing.